frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html

```diff
@@ -18,7 +18,7 @@ <h2>Project members</h2>
         <button
           (click)="openAddUserDialog()"
           mat-stroked-button
-          matTooltip="Add a new user"
+          [matTooltip]="'Add a new user'"
```

Comment:
```suggestion
          matTooltip="Add a new user"
```

I don't see a need for the change, it only leads to less performance.

Reply:
The change was there as I previously disabled the possibility of adding a user in an archived project. Simply forgot to also change this back.